tests/controller-test.js

```diff
@@ -244,6 +245,7 @@ OO = {
     Html5Skin.onPlayerCreated.call(controllerMock, 'customerUi', elementId, {skin:{config:{}}}, persistentSettings);
     Html5Skin.onSkinMetaDataFetched.call(controllerMock, 'customerUi', {});
     Html5Skin.onAttributesFetched.call(controllerMock, 'customerUi', {"attributes":{"provider":{"ots_stretch_to_output":"true"}}});
+    Html5Skin.trySetAnamorphicFixState.call(controllerMock, true);
```

Comment:
I had the same problem when writing unit tests for some Ooyala Ads changes. Can you check out the:

"describe('Controller testing Ooyala Ads', function () {"

section below. Would we be able to call the function and check the mock controller state like what was done in the 'Controller testing Ooyala Ads' tests?